Directory.Packages.props

```diff
@@ -11,7 +11,8 @@
   </ItemGroup>
   <ItemGroup Condition=" '$(IsTestProject)' != 'true' ">
     <PackageVersion Include="Microsoft.Extensions.Logging" Version="2.0.0" />
-    <PackageVersion Include="xunit.abstractions" Version="2.0.1" />
+    <PackageVersion Include="xunit.abstractions" Version="2.0.3" />
+    <PackageVersion Include="xunit.extensibility.execution" Version="2.4.1" />
```

Comment:
Are these the minimal possible versions that support the functionality?

Reply:
You are right, those are not the minimal possible versions (by a single minor version!). I addressed this in 0b75f6aad21fe72bece9e7a24c79fafd1caacc7c where I used the first versions to support .NET Standard 2.0.